src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Start using the new synthetic monitoring runtime
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+Synthetic monitoring has a new runtime that affects your public monitors. By converting from the legacy runtime to the new runtime, you improve the backend environment that your public monitors execute from. New synthetic monitors default to using the new runtime. If you've created a synthetic monitor using the legacy runtime, there are extra [configuration steps](#runtime-convert).
+
+Making the switch gives you these features:
+
+* Access to newer versions of Google Chrome, so your scripted browser monitors use browser versions similar to the ones used by your customers. 
+* Access to the updated scripted API runtime that uses `Got` node.js library  instead of the deprecated `Request` library. 
+* Support for async / await syntax. This simplifies script creation and maintenance, diverging from the old runtime’s `.then` promise chaining.
+* The ability for your monitors to treat objects like frames and find objects inside the shadow DOM.
+* [Custom timing library](SOMETHING) for scripted API monitors.
+- Access to automated management of your synthetic monitors using [NerdGraph APIs](SOMETHING). 
+
+We recommend converting your legacy runtime to take advantage of these capabilities. Eventually, we plan to make this new runtime available for private monitoring locations. 
+
+<Callout variant="important">
+Before switching, make sure that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Convert your monitors to the new runtime [#runtime-convert]
+
+To prevent your critical monitors from breaking during future end of life processes, we recommend you convert your public monitors sooner rather than later. To convert:
+
+1. Go to [one.newrelic.com](https://one.newrelic.com/nr1-core), click **Synthetics**, then select the monitor you want to edit.
+2. From the **Settings** tab, click **General**. 
+3. Use the dropdown menu to switch the current runtime view (use Node 16 for scripted APIs and Chrome 97 for scripted browser monitors). 
+4. Click **Validate** to check that your monitors function in the new runtime. Make any script script modifications if needed. 
+5. Save.
+
+## Switch to the legacy runtime from the UI [#runtime-UI]
```

Comment:
Is this switching to the legacy runtime or switching to the new runtime? We'll need to call out that the show new view switch is required here too.

Reply:
- Existing monitors use what it was configured as at the time (so if used Chrome 72, it would say Chrome 72)
- They would need to switch to new runtime From There